src/components/atoms/MapPreview.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="map">
+    <l-map :zoom="zoom" :center="center"
+      :options="{ zoomControl: false, scrollWheelZoom: false, dragging: false, doubleClickZoom: false }" ref="mapRef">
+      <l-tile-layer :url="url" :attribution="attribution"></l-tile-layer>
+      <l-marker :lat-lng="center"></l-marker>
+    </l-map>
+  </div>
+</template>
+
+<script>
+import "leaflet/dist/leaflet.css"
+import { LMap, LMarker, LTileLayer } from "@vue-leaflet/vue-leaflet";
+
+export default {
+  name: "MapPreview",
+  components: {
+    LMarker,
+    LTileLayer,
+    LMap,
+  },
+  props: {
+    zoom: {
+      type: Number,
+      required: true,
+    },
+    center: {
```

Comment:
I think only this should be a prop, move everything else to data

Reply:
True, but I think I'd keep both zoom and center as a prop though. The URL and attribution will be moved, that was carelessness of my part.